ming_sim/cli_backend.py

```
@@ -3410,8 +3410,18 @@ def _extract_secret_order(
         "  \"标签\": [\"相关人名/地区/事项关键词\"],\n"
         "  \"排除对象\": {\"人物\": [\"明确说要瞒住的人名\"], \"机构\": [\"不走的衙门\"]},\n"
         "  \"案卷关联\": [{\"目标案卷ID\": 123, \"类型\": \"护卫/稽核/接应\", "
-        "\"说明\": \"一句说明\"}]\n"
+        "\"说明\": \"一句说明\"}],\n"
+        "  \"差务\": \"差务类型名，如 补发饷银、缉获人犯、清丈；必须点明，不得用标题或标签猜测\",\n"
+        "  \"价值轴\": [\"闭集轴名，如 实务事功、既得利益\"],\n"
```

Ruff (0.16.2)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)